spacy/language.py

```diff
@@ -1018,6 +1018,98 @@ def __call__(
                 raise ValueError(Errors.E005.format(name=name, returned_type=type(doc)))
         return doc
 
+    def distill(
+        self,
+        teacher: "Language",
+        examples: Iterable[Example],
+        *,
+        drop: float = 0.0,
+        sgd: Optional[Optimizer] = None,
+        losses: Optional[Dict[str, float]] = None,
+        component_cfg: Optional[Dict[str, Dict[str, Any]]] = None,
+        exclude: Iterable[str] = SimpleFrozenList(),
+        annotates: Iterable[str] = SimpleFrozenList(),
+        component_map: Optional[Dict[str, str]] = None,
+    ):
+        """Update the models in the pipeline.
+        teacher (Language): Teacher to distill from.
+        examples (Iterable[Example]): A batch of examples
```

Comment:
Could we expand the description here to indicate that `reference` pertains to the `Doc` that's passed to the teacher (and `predicted` to the student)?

Reply:
Fixed. Also updated the docs for trainable pipes to mention this.